pkg/virt-api/api.go

```diff
@@ -754,6 +756,19 @@ func (app *virtAPIApp) Run() {
 	}
 }
 
+func (app *virtAPIApp) shouldEnableDebug() {
+	if app.clusterConfig.IsAPIDebugEnabled() {
+		if !app.isDebugEnabled {
+			log.Log.SetVerbosityLevel(5)
```

Comment:
`SetVerbosityLevel` and reading the verbosity level is right now not goroutine save. This will very likely end up in a mixed state where depending on which thread the goroutine runs it will log with different verbosity levels.